invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/ManagerMemberBulkActions.js

```diff
@@ -107,7 +111,7 @@ class ManagerMemberBulkActionsCmp extends Component {
 
   handleChooseCurrentAction = (value, selectedMembers, selectedCount) => {
     const serializeSelected = _mapValues(selectedMembers, ({ data }) => ({
-      ...data,
+      ...data.member,
```

Comment:
could you test if inviting members works fully after this change? fully = the whole workflow, up to a point when the user is invited